### DISCUSSION TITLE
NEW! Local Project Mode for Code 4 IBM i now available

### DISCUSSION THREAD

**Discussion by edmundreinhardt** [codefori/vscode-ibmi#1471] [General]
The IBM i Project Explorer extension is now available on both the Microsoft and OpenVSX registries.
https://marketplace.visualstudio.com/items?itemName=IBM.vscode-ibmi-projectexplorer

This extension to Code for IBM i  gives you the ability to do all of our development in local git projects and automatically synchronize with an IFS directory and build from there.  It reuses all of the Code for IBM i connections and functionality.

For those who want to have a fully VS Code native experience to editing and  source control for their IBM i application development, you should definitely check it out.

This supports BOB, make and ARCAD and others for build with integrated compile error feedback.

**Comment by beckhamrryyaann** [comment]
Looking forward to giving this a try!
